tests/jerry/regression-test-issue-122.js

```diff
@@ -15,10 +15,11 @@
 
 try
 {
-  print(this, []);
+  print({toString: function() { throw new TypeError("foo"); }}, []);
```

Comment:
You mean replace the whole `regression-test-issue-122.js` file with the simple print? But then we will not have a test case which creates an error for the `print` call.